requirements.txt

```diff
@@ -5,3 +5,4 @@ platformdirs
 bump2version
 filelock
 dnspython
+black
```

Comment:
Let's try this `black;python_version>="3.6"` so it won't fail tests in py35.

Reply:
@lipoja it works!